source/DG.HostApp/Pages/ClusterConfigManager.razor

```diff
@@ -0,0 +1,102 @@
+﻿@page "/clusterconfigeditor"
+
+@using DG.Core.ConfigManagers
+@using DG.Core.Models
+
+@inject IClusterConfigManager clusterConfigManager
+<h1>Cluster configuration manager</h1>
+<button class="btn btn-outline-primary" @onclick="AddNode">Add Node</button>
+<button class="btn btn-outline-primary" @onclick="AddApplicationInstance">Add Application instance</button>
+@if (clusterConfig == null)
+{
+    <p><em>Please, read the config file</em></p>
+}
+else
+{
+    <h2>Cluster configuration:</h2>
+    <p>
+        <b>Last update:</b> @clusterConfig.LastUpdateTime
+        <b>HashMd5:</b> @clusterConfig.HashMD5
+    </p>
+    <EditForm Model="@clusterConfig" OnValidSubmit="@WriteConfig">
+        <h3>Nodes:</h3>
```

Comment:
I like the idea to edit config file in form! I have a suggestion to divide view on two pages: separately to edit nodes and to edit instances, and add links between them, or something like this. This is a design moment to think about.

Reply:
I will create a new issue "Creating new better UI design for cluster config editor" .